aws_lambda_powertools/utilities/typing/lambda_cognito_identity.py

```diff
@@ -2,9 +2,7 @@
 
 
 class LambdaCognitoIdentity(object):
-    """
-    Information about the Amazon Cognito identity that authorized the request.
-    """
+    """Information about the Amazon Cognito identity that authorized the request."""
```

Comment:
This reminds me we need to prioritise a minter for NumPy docstrings

Reply:
@heitorlessa - sure